qa-dataset-ontology-schema/ontology.trig

```diff
@@ -0,0 +1,66 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl:  <http://www.w3.org/2002/07/owl#> .
+@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
+@prefix qa:   <https://www.statnett.no/Talk2PowerSystem/qa#> .
+@prefix dct:  <http://purl.org/dc/terms/> .
+
+<https://www.statnett.no/Talk2PowerSystem/qa#> {
+
+#################################################################
+# Ontology Definition
+#################################################################
+
+<https://www.statnett.no/Talk2PowerSystem/qa#> a owl:Ontology ;
+    rdfs:label "Talk2PowerSystem Q&A Dataset Ontology"@en ;
+    rdfs:comment "Ontology describing templates and paraphrases for the Q&A dataset in the Talk2PowerSystem project."@en ;
+    dct:creator "Graphwise.ai" ;
```

Comment:
Strictly speaking, `dct` is for resources (URLs) whereas `dc` is for resources or strings.
And there's no reason to use string for `creator` but URL for `publisher. In this case the two are the same and it's not a formal publication and the string says the same as the URL, so use just one:
```ttl
   dct:creator <https://graphwise.ai>
```

Reply:
Fixed for `cimr` as well